### PR TITLE
Add detect_errors command attribute to fasta_nucleotide_color_plot

### PR DESCRIPTION
**Pull request by gregvonkuster**
Also apply standard discussed here https://github.com/galaxy-iuc/standards/issues/43

### CODE REVIEW

File: tools/fasta_nucleotide_color_plot/fasta_nucleotide_color_plot.xml [hunk 1]
```diff
@@ -6,40 +6,38 @@
...
+    -C "$c_select_color_cond.c_rgb"
+#else
+    -C $c_select_color_cond.color_c
+#end if
+#if str($n_select_color_cond.n_select_color) == "n_rgb":
+    -N "$n_select_color_cond.n_rgb"
+#else
+    -N $n_select_color_cond.color_n
+#end if
+-o "$output" 1>/dev/null
```
Comment (bgruening) [inline tools/fasta_nucleotide_color_plot/fasta_nucleotide_color_plot.xml]: Single quotes :)

File: tools/fasta_nucleotide_color_plot/fasta_nucleotide_color_plot.xml [hunk 2]
```diff
@@ -6,40 +6,38 @@
...
-                -N "$n_select_color_cond.n_rgb"
-            #else
-                -N $n_select_color_cond.color_n
-            #end if
-            -o "$output" 1>/dev/null
-        ]]>
-    </command>
+    <command detect_errors="exit_code"><![CDATA[
+java -jar $__tool_directory__/FourColorPlot.jar
+-f "$input"
```
Comment (bgruening) [inline tools/fasta_nucleotide_color_plot/fasta_nucleotide_color_plot.xml]: single quotes, also above by $__tool__directory.

**Comment by gregvonkuster** [comment]
Oops!  These tools pre-date that, so fixed now.  Thanks @bgruening ;)

**Comment by nsoranzo** [comment]
Thanks @gregvonkuster!

**Pull request merged**